manual.asc

```diff
@@ -513,6 +513,21 @@ You can also manually import .apkg files as follows:
  * Choose the apkg file you just copied to your device when prompted
  * Tap OK
  
+== Importing Anki Databases (.anki2)
+ 
+AnkiDroid does not yet support directly importing Anki database (`.anki2`) files. A database import will replace your collection with the contents the provided file. If you want to perform a full replacement, you should ideally import an exported `collection[.apkg/.colpkg]`, as this also includes media data.
+ 
+=== Importing anki2 files manually
+
+`.anki2` files can manually be imported:
```

Comment:
```suggestion
This is not officially supported, but `.anki2` files can manually be imported if needed, in cases of troubleshooting for example:
```

Reply:
I'm fine with people shooting their foot off, I try to only use tools that allow it even, but expectations need to be clear - can't imagine trying to untangle the mess if someone does this and doesn't know what they are doing / are not okay with losing everything